tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/colorization_evaluator.py

```diff
@@ -374,8 +377,8 @@ def set_input_and_output(self):
 
 
 class ColorizationCheckModel(BaseModel):
-    def __init__(self, network_info, launcher, delayed_model_loading=False):
-        super().__init__(network_info, launcher, delayed_model_loading)
+    def __init__(self, network_info, launcher, net_type, delayed_model_loading=False):
```

Comment:
inside the model you know its type, so it can be constant field like in action recognition and text spotting, no need have it as constructor argument